rocketemu/default.nix

```diff
@@ -59,6 +59,8 @@ lib.mapAttrs
             SPIKE_INTERFACES_LIB_DIR = toLib spike_interfaces;
           };
 
+        buildFeatures = [ "trace" ];
+
```

Comment:
This is only used for debug, please remove it in the PR, we should have an additional nix PR for the trace and vcs features.